Shared/dugga.js

```diff
@@ -1100,29 +1100,12 @@ function AJAXService(opt,apara,kind)
 				success: returnedSection
 			});
 	}else if(kind=="PDUGGA"){
-		//for testing
-		console.log("dugga.js (kind=='PDUGGA')\n"+"courseid="+querystring['courseid']+"\ndid="+querystring['did']+"\ncoursevers="+querystring['coursevers']+"\nmoment="+querystring['moment']+"\nsegment="+querystring['segment']);
```

Comment:
We should add a switch for the opts here

Reply:
Commit 75eb68e contains the switch statements, i have also created a new issue "migrate showDuggaSevice.php" where implementing the switch statement is a part of the issue.